keyboards/specskeys/info.json

```
@@ -0,0 +1,15 @@
+        }
```

`LAYOUT_Fn` is not defined anywhere and should be removed. 
```suggestion
        }
```